index.js

```diff
@@ -247,12 +267,11 @@ const main = async () => {
   console.log('=================')
   console.log(`${total.toString().padStart(3, ' ')} Total ${getCount('total', ignoredCounts)}`)
 
-  const numFailingVulns =
-    failThreshold === 'informational' ? total - ignoredCounts.informational
```

Comment:
I'm not sure how to deal with `indeterminate`.
`indeterminate` is included in the `numFailingVulns` only when `failThreshold` equals `informational` because `total` includes `indeterminate`.

Reply:
I haven't seen any indeterminate vulnerabilities, they might exist only in concept. Let's go with your approach for now -- we can revise the code later if there are issues.